pkg/plugins/platform/local.go

```diff
@@ -291,6 +288,52 @@ func (l *LocalApp) SiteStatus() string {
 	return siteStatus
 }
 
+// Import performs an import from a local
+func (l *LocalApp) Import() error {
+	provider, err := l.AppConfig.GetProvider()
+	if err != nil {
+		return err
+	}
+
+	err = provider.Validate()
+	if err != nil {
+		return err
+	}
+
+	if l.SiteStatus() != SiteRunning {
+		util.Warning("Site is not currently running. Starting site before performing import.")
```

Comment:
I wasn't really thinking about it as "warning", more as "visible yellow text". Seemed worth informing the user we were starting the site.

That being said, I agree and am going to roll it back. We should probably doing things that would confuse our usage of warning/success/failed.